eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -1,5 +1,5 @@
 <!---
-   Copyright 2018 Jaden Young
+   Copyright 2017-2021 Ericsson AB and others.
```

Comment:
As commented in other places, this doesn't really match the original. Should we instead provide the copyright header as a value to a key with underscore in the yaml files?